Packs/IllusiveNetworks/Playbooks/playbook-Illusive-Retrieve-Incident.yml

```diff
@@ -0,0 +1,260 @@
+id: Illusive-Retrieve-Incident
+version: -1
+name: Illusive-Retrieve-Incident
+description: This playbook is used for retrieving an extensive view over a detected
+  incident by retrieving the incident details and a forensics timeline if and when
+  forensics have been successfully collected.
+starttaskid: '0'
```

Comment:
Playbooks also has outputs. Update the outputs of the playbook. Right now in these playbooks there are no defined outputs.
![image](https://user-images.githubusercontent.com/7270217/81586307-fbc03080-93bd-11ea-8e7d-550cf3d998d6.png)